pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+    - repo: https://github.com/python/black
+      rev: stable
+      hooks:
+          - id: black
+            language_version: python3.7
+    - repo: https://gitlab.com/pycqa/flake8
+      rev: 3.7.7
+      hooks:
+          - id: flake8
+            language: python_venv
```

Comment:
@TomAugspurger do you know if there is a difference taking flake8 from its own repo or from the pre-commit-hooks repo? 
In https://github.com/pandas-dev/pandas/issues/23616 you use the pre-commit-hooks one, but in a recent gitter chat you pasted the above?